integration-cli/docker_cli_build_test.go

```diff
@@ -669,6 +669,64 @@ func TestBuildCacheADD(t *testing.T) {
 	logDone("build - build two images with remote ADD")
 }
 
+func TestBuildLastModified(t *testing.T) {
+	name := "testbuildlastmodified"
+	defer deleteImages(name)
+
+	server, err := fakeStorage(map[string]string{
+		"file": "hello",
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.Close()
+
+	var out, out2 string
+
+	if _, out, err = buildImageWithOut(name,
+		fmt.Sprintf("FROM busybox\nADD %s/file /\nRUN ls -le /file", server.URL),
+		false); err != nil {
+		t.Fatal(err)
+	}
+
+	originMTime := regexp.MustCompile(`root.*/file.*\n`).FindString(out)
```

Comment:
originally I used 'stat' but as you said, busybox doesn't have it so I was left with just "ls" to get the timestamp.  I'm open to other ideas.  Why isn't the output an mtime string?  other stuff in there too but at long as it has the timestamp (with seconds) it shouldn't matter.